### PR TITLE
Add Environments and Biomes, Restructure Sectors

### PR DESCRIPTION
**Pull request by chatterchats**
* Add biomes and environments jsons.
* Updated Sectors.json to be an array vice dict.
* Updated planets.json to use these new and updated files.
* Fix trailing whitespace before dealloc yells at me

### CODE REVIEW

File: planets/environments.json
Comment (dealloc) [inline planets/environments.json:1]: perhaps this should be called `hazards`?

File: planets/planets.json
Comment (dealloc) [inline planets/planets.json:1]: perhaps this should be called `names.json`, since it's already in a folder called `planets`

**Review by dealloc — CHANGES_REQUESTED** [review]
solid work overall!
Aside from 2 minor comments on the filenames themselves, I see that you have converted `sectors` into an array, and same for biomes and environments.

I'd prefer to use an object with explicit keys (could be either a number or the name (which I favour)), since if new biomes get added, removed or the order changes this won't (implicitly) break other lists.

**Comment by chatterchats** [comment]
> solid work overall! Aside from 2 minor comments on the filenames themselves, I see that you have converted `sectors` into an array, and same for biomes and environments.
> 
> I'd prefer to use an object with explicit keys (could be either a number or the name (which I favour)), since if new biomes get added, removed or the order changes this won't (implicitly) break other lists.

It could go either way, my thought process was using indexes is slightly easier than having to use strings (python-coded) `biomes[biome_index] vs biomes[str(biome_index)]`

File: planets/environments.json
Comment (chatterchats) [reply planets/environments.json:1]: I was just using what the rest of the community was using so far, calling them environmentals, they're called hazards in-game so I could go either way

File: planets/planets.json
Comment (chatterchats) [reply planets/planets.json:1]: valid, change inc.

**Comment by dealloc** [comment]
> > solid work overall! Aside from 2 minor comments on the filenames themselves, I see that you have converted `sectors` into an array, and same for biomes and environments.
> > I'd prefer to use an object with explicit keys (could be either a number or the name (which I favour)), since if new biomes get added, removed or the order changes this won't (implicitly) break other lists.
> 
> It could go either way, my thought process was using indexes is slightly easier than having to use strings (python-coded) `biomes[biome_index] vs biomes[str(biome_index)]`
> 
> Similar thought process behind converting planets.json to an array so you can lookup by planet index as an int, vice a str.

I understand the reasoning, but I feel it's a little brittle and easy to break (especially when being reviewed in Githug PR rather than a full IDE).
You could easily just transform it into an array when reading it into Python if that's the preference, I feel like a keyed approach is 'safer' in terms of data integrity

File: planets/planets.json
Comment (dealloc) [inline planets/planets.json:1]: at this point the sector file feels a little redundant, should we get rid of it entirely?

**Review by dealloc — CHANGES_REQUESTED** [review]

File: planets/planets.json
Comment (chatterchats) [reply planets/planets.json:1]: Yes

**Review by dealloc — APPROVED** [review]

**Pull request merged**